public/app/plugins/datasource/grafana-azure-monitor-datasource/api/routes.ts

```diff
@@ -16,13 +16,13 @@ export function getManagementApiRoute(azureCloud: string): string {
 export function getLogAnalyticsManagementApiRoute(azureCloud: string): string {
```

Comment:
This method was created only because we had different credentials. Now we can delete it, and just use the `getManagementApiRoute` above.